.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ workflows:
 jobs:
   build-builder:
     docker:
-      - image: circleci/golang:1-stretch
+      - image: circleci/golang:1.13-stretch
```

Comment:
I *think* `golang:1-stretch` will present the current stable version (1.13 today, and 1.14 whenever that becomes stable).  If this got inherited from my test PR from a few weeks ago, I was just experimenting there.

Reply:
changeset ee45869 (from me) can be dropped / reverted